### ISSUE TITLE
Exception: FastRGF estimators are unavailable for usage

### ISSUE THREAD

**Issue by caprone** [HunterMcGushion/hyperparameter_hunter#106]
HI @HunterMcGushion ;)

then is fast implementation of RGF unavailable in Hunter??

**Comment by HunterMcGushion** [comment]
Great question! 

I had actually never tried using `FastRGFClassifier`, or `FastRGFRegressor`, but I just tested a slightly modified version of [HyperparameterHunter's RGF classification example](https://github.com/HunterMcGushion/hyperparameter_hunter/blob/master/examples/rgf_examples/classification.py), in which I change the two uses of `RGFClassifier` to `FastRGFClassifier`, and it works! 

There are two things to keep in mind, though:
1. Using the FastRGF estimators is, of course, completely dependent on having FastRGF properly built (not just PyPI’s “rgf-python”). I recommend seeing [their documentation](https://github.com/RGF-team/rgf/tree/master/FastRGF#2-installation) if you’re unsure about this
    - When testing that FastRGF is properly installed, I also recommend using it on a small toy problem without HyperparameterHunter. This way you can make sure FastRGF is set up correctly before adding HyperparameterHunter
2. `FastRGFClassifier` and `RGFClassifier` are not completely interchangeable because they have different parameters. So in the example linked above, I commented out the following parameters in both the `experiment` and call to `optimizer.set_experiment_guidelines`: “algorithm”, “normalize”, and “loss”
    - Specifically, `FastRGFClassifier` doesn’t have the parameters “algorithm” or “normalize”, and it expects different values for “loss”

I hope this helps! Please let me know if you have any other questions, so we can get this sorted out! Thanks for your question, again!

TL;DR: Yes, FastRGF does work with HyperparameterHunter, but it must be properly installed, and remember that its parameters differ from those of RGF’s normal estimators

**Comment by HunterMcGushion** [comment]
Glad to hear it! Thanks again for bringing up FastRGF!
